src/blocks/time.rs

```diff
@@ -30,6 +34,9 @@ pub struct TimeConfig {
     /// Update interval in seconds
     #[serde(default = "TimeConfig::default_interval", deserialize_with = "deserialize_duration")]
     pub interval: Duration,
+
+    #[serde(default = "TimeConfig::default_on_clicked")]
+    pub on_clicked: Option<String>,
```

Comment:
Can you change this to `on_click`, for consistency with the [Custom block](https://github.com/greshake/i3status-rust/blob/master/blocks.md#custom)?